mlir/utils/jenkins/Jenkinsfile

```diff
@@ -112,30 +112,32 @@ void preMergeCheckPackage(String codepath) {
 
 void postProcessPerfRes(String chip) {
     publishHTML (target: [
-        allowMissing: false,
+        allowMissing: ${chip=='gfx1100' ? true, false},
```

Comment:
`== gfx1100` is the wrong way to check for Navi3x. You probably want `.startswith("gfx11")`

Reply:
... never mind, thought this was the Python script